pkg/asset/machines/master.go

```diff
@@ -369,6 +369,18 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 			}
 		}
 
+		if mpool.OSImage.Publisher != "" {
```

Comment:
So, what happens when mpool.OSImage.Publisher == "" ? Can it be empty/unset? Looking at https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml#L350, it seems like a required field.

Reply:
We cannot remove it, since we need that value to retrieve the image with the API: https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/azure/client.go#L325-L334.

What would the INFO message accomplish in this case?